packages/commerce-sdk-react/src/auth/index.test.ts

```diff
@@ -340,8 +353,22 @@ describe('Auth', () => {
 
     test('ready - use refresh token when access token is expired with slas private client', async () => {
         const auth = new Auth(configSLASPrivate)
-        const JWTNotExpired = jwt.sign({exp: Math.floor(Date.now() / 1000) + 1000}, 'secret')
-        const JWTExpired = jwt.sign({exp: Math.floor(Date.now() / 1000) - 1000}, 'secret')
+        const JWTNotExpired = jwt.sign(
```

Comment:
Super minor non-blocking: I wonder if we can clean this up a bit so JWTNotExpired / JWTExpired isn't repeated so much in this class? Can we create these only once or have a function that creates these for us when needed?